docs/user/alerting/action-types/index.asciidoc

```diff
@@ -2,7 +2,7 @@
 [[index-action-type]]
 === Index action
 
-The index action type will index a document into {es}.
+The index action type will index a document into {es}. There is an {ref}/indices-create-index.html[info] how to create an index in {es}.
```

Comment:
Not sure what "There is an info how to create an index in Elasticsearch."  Should it be:

See also the [create index API](https://www.elastic.co/guide/en/elasticsearch/reference/current/indices-create-index.html).